src/api/queues.js

```diff
@@ -94,7 +87,20 @@ router.get(
   [requireQueue, failIfErrors],
   safeAsync(async (req, res, _next) => {
     const { id: queueId } = res.locals.queue
-    const queueResults = await Queue.findOne({
+    let queueResult = await Queue.findOne({
```

Comment:
We shouldn't have to re-query here - the queue is already accessible via `res.locals.queue` (see the `requireQueue` middleware).